### DISCUSSION TITLE
Campaigns - Send Email - Email Template

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#14763] [General]
In my 6.1 v in Campaigns module I could theoretically send email template to the list of Accounts (Email templates never shows).
Now in https://gitdeveloper.yetiforce.com/ I don't see this option. Is it removed or some error?

**Comment by rskrzypczak** [comment]
The option of sending e-mails is blocked on the demo version, so the button is unavailable.
You should have added an email template for the campaign module, if you have one, check this fix https://github.com/YetiForceCompany/YetiForceCRM/commit/bcb081fea937f36a7cbab422542881e28823c6cc
Otherwise, you just don't have permission to email templates.

**Reply by s2b-git** [reply]
thank you
